src/components/NcUserBubble/NcUserBubble.vue

```diff
@@ -250,6 +289,10 @@ export default {
 			return true
 		},
 
+		isClickable() {
+			return !!this.$listeners?.click || !this.popoverEmpty
```

Comment:
```suggestion
			return this.isLinkComponent === 'a'
```

Reply:
This skips `RouterLink`. but we can use `a` as a selector directly in CSS

Reply:
> This skips RouterLink. but we can use a as a selector directly in CSS

True, probably even easier. (BTW just adjusted the suggestion: Invert the condition ;) )